### ISSUE TITLE
v0.19.0 Responses without `:status` throw nullpointer exception

### ISSUE THREAD

**Issue by M-sleeper** [sunng87/ring-jetty9-adapter#95]
If the handler returns a response map without the key `:status` or a plain string an exception will be thrown. These were supported before with the default status of 200.
The websocket connection upgrader function throws the exception:
```clojure
(defn websocket-upgrade-response?
  [{:keys [^long status ws]}]
  ;; NOTE: we know that when :ws attr is provided in the response, we
  ;; need to upgrade to websockets protocol.
  (and (== 101 status) ws))
```

The `==` needs a number and with the typehint it tries to handle `nil` as long, check:
```clojure
(let [^long a nil]
    (== a 111))
```
I suggest to update the condition to one of these:
```clojure
(and status (== 101 status) ws))
(and ws status (== 101 status)))
(and ws (== 101 status))) ;; status is needed when ws is present
```

**Comment by sunng87** [comment]
Thank you for reporting @M-sleeper. I will create a fix for this.

**Comment by sunng87** [comment]
Fixed in 63bccfa8b4957e075d38a0879a7a2846af9aabaa

**Comment by M-sleeper** [comment]
@sunng87 Thank you!
